k8s/wordpress/README.md

```diff
@@ -191,7 +229,35 @@ an external IP address for your WordPress app, run the following command:
 ```
 kubectl patch svc "$APP_INSTANCE_NAME-wordpress-svc" \
   --namespace "$NAMESPACE" \
-  --patch '{"spec": {"type": "LoadBalancer"}}'
+  --patch '{"spec": {"type": "NodePort"}}'
+
+APPLICATION_UID=$(kubectl get applications/${APP_INSTANCE_NAME} --namespace="$NAMESPACE" --output=jsonpath='{.metadata.uid}')
```

Comment:
nit - please make the variables expansion consistent, with either ${var} or $var pattern